src/mds/MDSRank.cc

```diff
@@ -3149,7 +3149,7 @@ void MDSRankDispatcher::evict_clients(
   dout(20) << __func__ << " matched " << victims.size() << " sessions" << dendl;
 
   if (victims.empty()) {
-    on_finish(-ESRCH, "no hosts match", outbl);
+    on_finish(0, {}, outbl);
```

Comment:
Any reason to not also say "no hosts match"?

Reply:
That's acceptable too.

Reply:
Fixed.